cmd/ova-provider-server/nfs_test.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/onsi/gomega"
```

Comment:
```suggestion
	. "github.com/onsi/gomega"
```
and then we can drop `gomega` below